packages/flutter/test/material/chip_test.dart

```diff
@@ -1912,6 +1912,36 @@ void main() {
     await gesture.removePointer();
   });
 
+  testWidgets('Chips with background color should have delete icon ripple and splash in the foreground', (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/41461
+    final Key deleteKey = UniqueKey();
+    await tester.pumpWidget(
+      _wrapForChip(
+        child: Chip(
+          onDeleted: () { },
+          backgroundColor: Colors.pink,
+          label: const Text('Chip with delete icon'),
+          deleteIcon: Icon(Icons.cancel, key: deleteKey),
+        ),
+      ),
+    );
+    final Offset center = tester.getCenter(find.byKey(deleteKey));
+
+    final TestGesture gesture = await tester.startGesture(center);
+    await tester.pump(); // start gesture
+    await tester.pump(const Duration(milliseconds: 200)); // wait for splash to be well under way
+    final RenderBox box = Material.of(tester.element(find.byType(InkResponse))) as dynamic;
```

Comment:
This is pretty strange, in part because `Material.of()` returns a renderer, but it's consistent with ink_paint_test.dart. OK.

Reply:
Yes, I did the same as the ink_paint_test.dart here.